app/javascript/lib/metrics-api.js

```diff
@@ -123,7 +123,7 @@ export function logClickLink(target) {
     id: target.id
   }
   // Check if target is a tab that's not a part of a menu
-  const parentTabList = $(target).closest("[data-analytics-name][role='tablist']")
+  const parentTabList = $(target).closest('[data-analytics-name][role=\'tablist\']')
```

Comment:
ESLint substantially degrades readability in this case, so let's disable it here.

```suggestion
  // eslint-disable-next-line
  const parentTabList = $(target).closest("[data-analytics-name][role='tablist']")
```